api/app/signals/apps/signals/migrations/0154_statusmessagetemplate_is_active.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-License-Identifier: MPL-2.0
+# Copyright (C) 2022 Gemeente Amsterdam
+# Generated by Django 3.2.12 on 2022-03-09 07:07
+
+from django.db import migrations, models
+
+
+def _update_existing_status_message_templates(apps, schema_editor):
+    """
+    All StatusMessageTemplates that already exists shoudl have te is_active set to True
+    """
+    StatusMessageTemplate = apps.get_model('signals', 'StatusMessageTemplate')
+    StatusMessageTemplate.objects.all().update(is_active=True)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('signals', '0153_drop_experimental_signals_filter_view'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='statusmessagetemplate',
+            name='is_active',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.RunPython(_update_existing_status_message_templates, lambda *args, **kwargs: None),
```

Comment:
Nope you cannot 😉 because than this migration is not reversible. And executing `python manage.py migrate signals 0153` will raise an `django.db.migrations.exceptions.IrreversibleError`